reliability-v2/tasks/Tasks.py

```diff
@@ -330,7 +349,12 @@ def kubectl_task(self,task,user):
 
     def shell_task(self,task,user):
         kubeconfig = global_data.kubeconfigs[user]
-        (result, rc) = shell(task, kubeconfig)
-        self.__log_result(rc)
+        (result, rc) = shell(task, kubeconfig, user)
+        start_index=task.rfind('/')
```

Comment:
Can we add a comment here or above each function start with some explanation of the intent of the function?

Reply:
Added more comments to the functions